java/src/main/java/com/google/oak/verification/MainVerifier.java

```diff
@@ -38,11 +39,12 @@ public static void main(String[] args) throws Exception {
   public MainVerifier(Evidence evidence, Endorsements endorsements) {
     this.evidence = evidence;
     this.endorsements = endorsements;
-    if (evidence.getLayersCount() == endorsements.getLayersCount()) {
+    if (evidence.getLayersCount() - 1 == endorsements.getCustomLayersCount()) {
```

Comment:
Maybe add a comment explaining the off-by-one?

Reply:
Done.